### ISSUE TITLE
Versioning labels

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tamzi/bungeni-portal#626]
```
- Some workflow transition give this generic and not helpful message «New 
version created upon workflow transition. » we need to give more meaningful 
messages

- in other workflow transistion we have eg. «New version created upon 
submission to clerks office » or «New Version created upon approval by 
speakers office » could be more informative/useful if phrase like this … I 
think …  e.g. <«Version submitted to clerks office » «Version approved by 
the Speakers Office»

- In short it  when a version is create take a snapshot of the latest version 
before the new «status» … this simple fact should be simply stated :-)
```

Original issue reported on code.google.com by `ashok.ha...@gmail.com` on 15 Jun 2010 at 8:35
